packages/sensors/android/src/main/java/io/flutter/plugins/sensors/SensorsPlugin.java

```diff
@@ -15,7 +15,9 @@
 /** SensorsPlugin */
 public class SensorsPlugin implements EventChannel.StreamHandler {
   private static final String ACCELEROMETER_CHANNEL_NAME = "plugins.flutter.io/accelerometer";
+  private static final String GRAVITY_CHANNEL_NAME = "plugins.flutter.io/gravity";
   private static final String GYROSCOPE_CHANNEL_NAME = "plugins.flutter.io/gyroscope";
+  private static final String LINEAR_ACCELEROMETER_CHANNEL_NAME = "plugins.flutter.io/linear";
```

Comment:
The channel names must be unique across all plugins. That is probably OK for `gyroscope`, while `linear` is at risk of name collisions. I suggest changing all of them so that the use the plugin name as a name space. So we'd have `plugins.flutter.io/sensors/gravity` etc.